lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -267,6 +273,31 @@ class PolygonPainter extends CustomPainter {
     }
 
     drawPaths();
+
+    if (polygonLabels && drawLabelsLast) {
+      for (final polygon in polygons) {
+        final offsets = getOffsets(polygon.points);
+        if (offsets.isEmpty) {
+          continue;
+        }
+
+        if (polygon.label != null) {
+          final painter = buildLabelTextPainter(
+            polygon.points,
+            polygon.labelPosition,
+            placementPoint: map.getOffsetFromOrigin(polygon.labelPosition),
+            points: offsets,
+            labelText: polygon.label!,
+            labelStyle: polygon.labelStyle,
+            rotationRad: map.rotationRad,
+            rotate: polygon.rotateLabel,
+            padding: 50,
```

Comment:
This `padding` is different from the `padding: 10` in the other `builtLabelTextPainter` call.

Reply:
Good catch. It wasn't intentional.

I picked a bigger value for my app and 50 is what I settled on. Not sure if there's a single right value.

For context, the padding says at what point the label should go away when you zoom out. Specifically, the polygon gets smaller but the label keeps it size, so at some point when the label plus padding doesn't fully fit into the polygon anymore the code drops it. I found that a small padding will keep the label longer on screen but may make it look a bit crammed :shrug:

Reply:
Maybe we should go with something like 20 then, as I do remember seeing 10 as a little cramped.

Reply:
Happy to change, no strong feelings around any particular value.

Since you brought this up, I also remembered that I had bumped the "maxLines" from 1 to 2 for my use-case. Arguably the value is very use-case dependend, would you like me to add a configurable parameter or simply not limit the lines?